src/lib/y2network/autoinst/interfaces_reader.rb

```diff
@@ -88,56 +87,81 @@ def load_generic(config, interface_section)
         config.name = name_from_section(interface_section)
         config.interface = config.name # in autoyast name and interface is same
         if config.bootproto == BootProtocol::STATIC
-          if !interface_section.ipaddr
+          # we need at least some ip for the static configuration
+          if !interface_section.ipaddr && !interfaces_section.aliases
```

Comment:
interface_section attributes are empty strings, arent them? except aliaes that is an empty hash, and interfaces_section looks like a typo

Reply:
no. If you don't put e.g. ipaddr into the profile, you get nil here.

Reply:
This is not correct ipaddr is an empty string and aliases will be an empty hash, in fact the second condition is never checked and that is why a different exception is not raised. But this is wrong.

See https://github.com/yast/yast-network/blob/master/src/lib/y2network/autoinst_profile/interface_section.rb#L268

Reply:
In fact, already described in this PR https://github.com/yast/yast-network/pull/1035